src/modules/auth/index.js

```diff
@@ -50,6 +52,18 @@ export default class SeaCatAuthModule extends Module {
 			name: 'New resource',
 			component: ResourcesCreateContainer
 		});
+		app.Router.addRoute({
+			path: '/auth/resources/deleted',
```

Comment:
@petrKavulok I am still thinking, if `'/auth/resources/!deleted'` would not be better. It helps to distinguish between resource id and the action (in this case delete) as you can see in `New resource` part of the app (ResourcesCreateContainer)